### PR TITLE
TY-4232 fcm migration

### PR DESCRIPTION
**Pull request by zpeng01**

### CODE REVIEW

File: src/gcm_request.erl [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 -define(BASEURL, "https://fcm.googleapis.com/fcm/send").
 -define(PROJECT_BASEURL, "https://fcm.googleapis.com").
--define(PROJECT_SEND_METHOD, "message:send").
+-define(PROJECT_SEND_METHOD, "messages:send").
```
Comment (TT-lwang) [inline src/gcm_request.erl:8]: why change this MACRO?

File: src/gcm_request.erl [hunk 2]
```diff
@@ -55,8 +86,6 @@ json_post_request(BaseUrl, Headers, Body) ->
     {ok, {{_, Code, _}, _, _}} ->
       lager:warn("Unknown error: ~p~n", [Code]),
       {http_error, Code};
-    {ok, {{StatusLine, _, _}, _, _Body}} ->
```
Comment (TT-lwang) [inline src/gcm_request.erl:58]: Why remove this?

File: src/gcm_request.erl (hunk 1 above)
Comment (zpeng01) [reply src/gcm_request.erl:8]: the new api URL is *messages:send...

File: src/gcm_request.erl (hunk 2 above)
Comment (zpeng01) [reply src/gcm_request.erl:58]: the case clause is covered by line 86.

**Review by TT-lwang — APPROVED** [review]

**Pull request merged**